### DISCUSSION TITLE
Thank You first off....Can we see temps in Fahrenheit instead of Celsius on the taskbar icon by chance? Just a preference of mine is all.

### DISCUSSION THREAD

**Discussion by faslanetech** [seerge/g-helper#1620] [General]
This is an amazing lil helper ;-)   Thank you!

I have it all configured on my M16 2121 model and uninstalled all the Armoury Crate bloat and this is awesome!
Is there a way to change the Celsius temp to Fahrenheit? I didn't see it in the app anywhere and read the FAQ and didn't see a reference on how/if it could be done in the .json file or not. Thanks a million regardless. PayPal coming your way!

Fas

**Comment by seerge** [comment]
@faslanetech hello. Currently you can't do that.

Isn't Celsius an industry standard for PC temperature measurement, even in US ? :)

**Reply by faslanetech** [reply]
U . S is solely Fahrenheit more or less. Not everyone but I bet 90% is.. It varies.. Why most software system monitor apps have the option to choose either one. Just a preference based on demographics :-)

**Comment by faslanetech** [comment]
If you were raised using the metric system you use Celcius, otherwise you'd use Fahrenheit.
It's just a demographic thing. The U. S. for instance uses Fahrenheit, Miles per hour and 12hr time whereas Europe uses Celcius, Kilometer per hour and 24hr time. So I was just curious if there was a setting somewhere. It's all about the metric system. The U .S. is ass backwards. Haha



https://science.howstuffworks.com/nature/climate-weather/atmospheric/us-use-fahrenheit.htm


Thank you!
L

________________________________
